src/features/components/atoms/card/card-primary/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import Typography from '../../typography'
+import Flex from '../../flex-box'
+import Link from '../../link'
+import Image from '../../image'
+import { Localize } from 'components/localization'
+import { TString } from 'types/generics'
+import dclsx from 'features/utils/dclsx'
+import useBreakpoints from 'components/hooks/use-breakpoints'
+
+type CardItemsProps = {
+    header: TString
+    description: TString
+    icon?: string
+    link?: `/${string}`
+}
+
+const CardPrimary = ({ header, description, icon, link }: CardItemsProps) => {
+    const { is_mobile_or_tablet } = useBreakpoints()
+
+    return (
+        <Flex.Box
+            className={dclsx('item_container')}
```

Comment:
Hi @meenakshi-deriv just asking do need to import this class `item_container` ?  
Thanks @meenakshi-deriv 🙏